build/Common.props

```diff
@@ -43,6 +43,7 @@
     <OpenTelemetryCoreLatestVersion>[1.9.0,2.0)</OpenTelemetryCoreLatestVersion>
     <OpenTelemetryCoreLatestPrereleaseVersion>[1.9.0-rc.1]</OpenTelemetryCoreLatestPrereleaseVersion>
     <StackExchangeRedisPkgVer>[2.1.58,3.0)</StackExchangeRedisPkgVer>
+    <ConfluentKafkaPkgVer>[2.3.0,3.0)</ConfluentKafkaPkgVer>
```

Comment:
not a blocker - this could be bump to 2.4.0 now?

Reply:
@vishweshbankwar  Indeed 2.4.0 has been released since I started working on this. I will push the package update in one of the planned follow up PR.